core/ui/src/main/java/org/phoebus/ui/javafx/Tracker.java

```diff
@@ -179,7 +179,7 @@ void hookEvents()
         // Keep the keyboard focus to actually get key events.
         // The RTImagePlot will also listen to mouse moves and try to keep the focus,
         // so the active tracker uses an event filter to have higher priority
-        tracker.addEventFilter(MouseEvent.MOUSE_MOVED, event ->
+        tracker.addEventFilter(MouseEvent.MOUSE_CLICKED, event ->
```

Comment:
This change I am not 100% sure about. Previously (i.e., with `MouseEvent.MOUSE_MOVED`), the selected items in the Display Editor were given focus whenever the mouse hovered over them. I found this behavior counter-intuitive, and with this change, they are instead given the focus by clicking on the selection using the mouse.